File: OPENAPI-REST-API/swagger-client/dart/lib/model/delete_geo_firewall_rule.dart
part of swagger.api;

class Delete Geo Firewall Rule {
  
  int ruleId = null;

  Delete Geo Firewall Rule();

  @override
  String toString() {
    return 'Delete Geo Firewall Rule[ruleId=$ruleId, ]';
  }

  Delete Geo Firewall Rule.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    ruleId = json['rule_id'];
  }

  Map<String, dynamic> toJson() {
    return {
      'rule_id': ruleId
     };
  }

  static List<Delete Geo Firewall Rule> listFromJson(List<dynamic> json) {
    return json == null ? new List<Delete Geo Firewall Rule>() : json.map((value) => new Delete Geo Firewall Rule.fromJson(value)).toList();
  }

  static Map<String, Delete Geo Firewall Rule> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, Delete Geo Firewall Rule>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new Delete Geo Firewall Rule.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/DeleteGeoFirewallRule.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * Delete geo firewall rule for your scrub ip
 */




public class DeleteGeoFirewallRule   {
  private Integer ruleId = null;

  public DeleteGeoFirewallRule ruleId(Integer ruleId) {
    this.ruleId = ruleId;
    return this;
  }

  /**
   * Get ruleId
   * @return ruleId
   **/
    public Integer getRuleId() {
    return ruleId;
  }

  public void setRuleId(Integer ruleId) {
    this.ruleId = ruleId;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    DeleteGeoFirewallRule deleteGeoFirewallRule = (DeleteGeoFirewallRule) o;
    return Objects.equals(this.ruleId, deleteGeoFirewallRule.ruleId);
  }

  @Override
  public int hashCode() {
    return Objects.hash(ruleId);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class DeleteGeoFirewallRule {\n");
    
    sb.append("    ruleId: ").append(toIndentedString(ruleId)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}
